xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -899,11 +1345,30 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="destination">The buffer in which to write.</param>
+        <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.</param>
+        <summary>Reads the next data item as a byte string (major type 2).</summary>
+        <returns>
+          <see langword="true" /> and advances the reader if <paramref name="destination" /> had sufficient length to receive the value, otherwise <see langword="false" /> and the reader does not advance.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="destination" /> had sufficient length to receive the value and the reader advances; otherwise, <see langword="false" />.</returns>
```